### PR TITLE
Clean up language

### PR DESCRIPTION
**Pull request by elharo**
## Description
Make sentences less awkward

## Motivation and Context
docs matter

## Impact
none

## Test Plan
sandcastle

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 1]
```diff
@@ -140,11 +140,11 @@ will function as both a coordinator and worker, use this configuration:
     discovery-server.enabled=true
     discovery.uri=http://example.net:8080
 
-If single coordinator is not sufficient, disaggregated coordinator setup can be used which supports multiple coordinator using below minimal configuration:
+If a single coordinator is not sufficient, disaggregated coordinator setup can be used which supports multiple coordinators using the following minimal configuration:
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/installation/deployment.rst:143]:
```suggestion
If a single coordinator is not sufficient, use a disaggregated coordinator setup which supports multiple coordinators using the following minimal configuration:
```
Nit to add "a" here as well, suggest change to active voice.

**Review by steveburnett — COMMENTED** [review]
Nice catches, thanks! One nit of a suggestion.

File: presto-docs/src/main/sphinx/installation/deployment.rst (hunk 1 above)
Comment (elharo) [reply presto-docs/src/main/sphinx/installation/deployment.rst:143]: done

**Review by steveburnett — APPROVED** [review]
LGTM! (docs)

Pulled updated branch, new local doc build, looks good.

**Pull request merged**